cocos/particle/particle-culler.ts

```diff
@@ -99,23 +99,23 @@ export class ParticleCuller {
     }
 
     private _initModuleList (): void {
-        _anim_module.forEach((val): void => {
-            const pm = this._particleSystem[val];
+        _anim_module.forEach((val: string): void => {
+            const pm = this._particleSystem[val] as IParticleModule;
             if (pm && pm.enable) {
                 if (pm.needUpdate) {
-                    this._updateList[pm.name] = pm;
+                    this._updateList.set(pm.name, pm);
```

Comment:
_updateList is a Map<string, ...>, Map should use `set/get` to store and read elements.